src/views/StandaloneOfficeAddressFiling.vue

```diff
@@ -866,7 +866,7 @@ export default class StandaloneOfficeAddressFiling extends Mixins(CommonMixin, D
     // fetch original office addresses with new date + update working data
     // (this will overwrite the current data)
     this.isFetching = true
-    if (!this.isJestRunning) {
+    if (!this.isVitestRunning) {
```

Comment:
Boy, some day we should just fix the unit tests so we can rid of this "test code" in Prod.

Reply:
Agreed.